### PR TITLE
Raise error in CameraReadout.from_name if no corresponding file can be found

### PR DESCRIPTION
**Pull request by maxnoe**
* Raise error in CameraReadout.from_name if requested data is not available
* Set `readout` to None in `CameraDescription.from_name` if that exception is raised.

### CODE REVIEW

File: ctapipe/instrument/camera/readout.py [hunk 1]
```diff
@@ -111,25 +111,11 @@ def from_name(cls, camera_name="NectarCam", version=None):
...
-            )
-            reference_pulse_shape = np.array([norm.pdf(np.arange(96), 48, 6)])
-            return cls(
-                camera_name=camera_name,
-                sampling_rate=u.Quantity(1, u.GHz),
-                reference_pulse_shape=reference_pulse_shape,
-                reference_pulse_sample_width=u.Quantity(1, u.ns),
-            )
+        tabname = "{camera_name}{verstr}.camreadout".format(
+            camera_name=camera_name, verstr=verstr
```
Comment (nbiederbeck) [inline ctapipe/instrument/camera/readout.py:115]: maybe use f-string syntax?

File: ctapipe/instrument/camera/tests/test_readout.py [hunk 2]
```diff
@@ -145,5 +145,8 @@ def test_hashing():
 @pytest.mark.parametrize("camera_name", camera_names)
 def test_camera_from_name(camera_name):
     """ check we can construct all cameras from name"""
-    camera = CameraReadout.from_name(camera_name)
-    assert str(camera) == camera_name
+
+    # these two don't have readout definitions on the dataserver
+    if camera_name not in ["MAGICCam", "Whipple109"]:
```
Comment (nbiederbeck) [inline ctapipe/instrument/camera/tests/test_readout.py:150]: should we test one of those? I think we only test LSTCam (at least on my machine), which will then not really test the new behaviour introduced in this PR

File: ctapipe/instrument/camera/readout.py (hunk 1 above)
Comment (maxnoe) [reply ctapipe/instrument/camera/readout.py:115]: I didn't add/change this code, just removed a try except around. There is no real reason to remove format

File: ctapipe/instrument/camera/tests/test_readout.py (hunk 2 above)
Comment (maxnoe) [reply ctapipe/instrument/camera/tests/test_readout.py]: done

**Review by nbiederbeck — APPROVED** [review]

**Review by LukasNickel — APPROVED** [review]

**Pull request merged**